docs/blog/2019-05-28-first-blog-post.md

```diff
@@ -0,0 +1,9 @@
+---
+slug: introducing-dewy
```

Comment:
File name seems wrong -- maybe update the date and call it `<date>-introducing-dewy.md` (to reflect the slug?